ephios/plugins/basesignup/signup/section_based.py

```diff
@@ -233,16 +233,37 @@ def get_configuration_fields(self):
             },
         }
 
-    def get_participant_count_bounds(self):
-        return (
-            sum(section.get("min_count") or 0 for section in self.configuration.sections),
-            sum(
-                section.get("max_count") or section.get("min_count")
-                for section in self.configuration.sections
-            )
-            if all(section.get("max_count") for section in self.configuration.sections)
-            else None,
+    def get_signup_stats(self):
+        from ephios.core.signup.methods import SignupStats
+
+        participations = self.shift.participations.filter(
+            state=AbstractParticipation.States.CONFIRMED
         )
+        requested_count = self.shift.participations.filter(
+            state=AbstractParticipation.States.REQUESTED
+        ).count()
+        signup_stats = SignupStats(requested_count, 0, None, 0)
+        for section in self.configuration.sections:
+            participation_count = participations.filter(
+                data__dispatched_section_uuid=section["uuid"]
+            ).count()
+            missing = (
+                max(section.get("min_count") - participation_count, 0)
+                if section.get("min_count")
```

Comment:
```suggestion
                if (min_count := section.get("min_count"))
```

and than use `min_count` in the line above.